src/config/config.ts

```diff
@@ -31,3 +32,20 @@ export const API = {
 
 /* default refresh token interval: 1 hour and 50 minutes; JEXIA tokens expire in 2 hours */
 export const DELAY = 1000 * 60 * 110;
+
+/**
+ * The default project zone
+ */
+export const DEFAULT_PROJECT_ZONE = "nl00";
```

Comment:
Do you really need to export default project zone?

Reply:
yes, for testing purposes 😇